test/include_generator.jl

```diff
@@ -29,6 +25,33 @@ n = rand(1:8)
 @test "randsym" in MatrixDepot.mdlist(:random)
 @test "randsym" in MatrixDepot.mdlist(:symmetric)
 
-import MatrixDepot: include_generator, Group
-@test_throws ArgumentError include_generator(Group, :lkjasj, sin)
+@test_nowarn MatrixDepot.init()
+n = rand(1:8)
+@test matrixdepot("randsym", n) !== nothing
+@test mdinfo("randsym") != nothing
+@test "randsym" in MatrixDepot.mdlist(:random)
+@test "randsym" in MatrixDepot.mdlist(:symmetric)
+
+begin #Testing backward compatibility deprecation. Delete eventually.
```

Comment:
This was resolved in https://github.com/peterahrens/MatrixDepot.jl/pull/79/commits/4b07108d4225dbcccef7deaa0ce235a617abd9f9. There was some complicated code to avoid issuing warnings for users that do not customize the old default files in `MYDEPOT`.